waiter/src/waiter/scheduler/marathon.clj

```diff
@@ -560,8 +565,8 @@
 (defn marathon-scheduler
   "Returns a new MarathonScheduler with the provided configuration.
    Validates the configuration against marathon-scheduler-schema and throws if it's not valid."
-  [{:keys [home-path-prefix http-options force-kill-after-ms framework-id-ttl mesos-slave-port
-           slave-directory sync-deployment url
+  [{:keys [authorizer home-path-prefix http-options force-kill-after-ms framework-id-ttl
```

Comment:
Should we add schema validation for `authorizer` in the pre-condition?

Reply:
Done.